server/methods/projects.js

```diff
@@ -37,5 +38,14 @@ Meteor.methods({
   'projects/sync': function () {
     if (!this.userId) throw new Meteor.Error('Only a logged in user can sync projects')
     return syncAllAsync(this.userId)
+  },
+
+  'projects/tag': function (projectId) {
+    if (!this.userId) throw new Meteor.Error('Only a logged in user can tag a project')
+
+    let project = Projects.findOne(projectId)
+    if (!project) throw new Meteor.Error('Cannot find project')
+
```

Comment:
Should we check that this user has access to that project?  There is an array in the [project doc](https://github.com/tableflip/the-platform-client/blob/master/lib/imports/schemas/project.js#L25).